### ISSUE TITLE
Improve the pravega benchmark tool logging

### ISSUE THREAD

**Issue by deenav** [pravega/pravega-benchmark#38]
When executing pravega benchmark tool, log/error messages being sent to STDOUT doesn't have valid timestamp attached.  If there is an IO failure/Exceptions, it is difficult to debug the logs at later point of time. And timestamp would be very helpful to find out what went wrong when compare it with pravega server logs/errors,.

Example logs from Pravega-benchamrk 
```[clientInternal-4] INFO io.pravega.client.segment.impl.SegmentOutputStreamImpl - Establishing connection to PravegaNodeUri(endpoint=172.24.21.14, port=12345) for Scope/_RGnew-test5/0.#epoch.0, writerID: 607ec962-a281-4972-ac7c-46d8302a69d4
[epollEventLoopGroup-6-6] INFO io.pravega.client.segment.impl.SegmentOutputStreamImpl - Received AppendSetup WireCommands.AppendSetup(type=APPEND_SETUP, requestId=1, segment=Scope/_RGnew-test5/0.#epoch.0, writerId=74f0efc4-85a0-4faa-aae6-e77902d39345, lastEventNumber=-9223372036854775808)
[epollEventLoopGroup-6-6] INFO io.pravega.client.segment.impl.SegmentOutputStreamImpl - Connection setup complete for writer 74f0efc4-85a0-4faa-aae6-e77902d39345
[epollEventLoopGroup-5-2] INFO io.pravega.client.netty.impl.ClientConnectionInboundHandler - Connection established ChannelHandlerContext(ClientConnectionInboundHandler#0, [id: 0xc542671d]) 
[epollEventLoopGroup-5-3] INFO io.pravega.client.netty.impl.ClientConnectionInboundHandler - Connection established ChannelHandlerContext(ClientConnectionInboundHandler#0, [id: 0x010919a1]) 
 ```

**Comment by RaulGracia** [comment]
Thanks @deenav for opening this issue. Apart from timestamps, we also need to include some informative log lines across the application to understand its execution workflow (i.e., when a ReaderGroup is instantiated, when a reader is closed, etc.). This issue could be more broad, and target general improvements for logging in the benchmarking tool.

**Comment by kmgowda** [comment]
These messages are produced by the pravega client. This bug should be raised on the pravega client.

**Comment by RaulGracia** [comment]
Fixed in #77.
